### DISCUSSION TITLE
Do the interactive modules get turned in?

### DISCUSSION THREAD

**Discussion by EasternOT** [cooperrc/computational-mechanics#217] [Q&A] [answered]
Looking through google classroom i see that there are due dates for the HW's and project's portion of the modules but does the work that we do in the interactive modules need to be submitted as well? I've been just following through the interactive modules via my jupyter workspace, not sure if that's right or wrong?

**Comment by cooperrc** [comment, marked as answer]
The only notebooks that need to be turned in are HWs and projects. 

The reading exercises are a great way to get practice and background info on how to complete the HW+project in each module.
